### PR TITLE
⬆️ UPGRADE: Relax pin to allow sphinx v4

### PR DESCRIPTION
**Pull request by hoefling**
Signed-off-by: oleg.hoefling <oleg.hoefling@gmail.com>

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,16 +11,17 @@
 # then then deleting compiled files has been found to fix it: `find . -name \*.pyc -delete`
 
 [tox]
-envlist = py37-sphinx3
+envlist = py39-sphinx4
```
Comment (chrisjsewell) [inline tox.ini:14]:
```suggestion
envlist = py38-sphinx4
```

**Review by chrisjsewell — APPROVED** [review]
cheers!

**Comment by hoefling** [comment]
@chrisjsewell yay, thank you! Will you release a new version of `sphinx-panels`? The 0.5.2 does not allow to be installed alongside Sphinx==4. This was the main reason for this simple PR :blush:

**Comment by alliter8** [comment]
I came here looking for this exact thing; thank you very much! I will be looking for the updated version soon. We are using Sphinx>=4.0.0, so this upgrade is essential if we want to keep using sphinx-panels, which we do. :D HOORAY!! You @hoefling are amazing! 🥳

**Pull request merged**